db/collection.go

```diff
@@ -13,10 +13,13 @@ package db
 import (
 	"bytes"
 	"context"
+	"encoding/json"
 	"fmt"
+	"reflect"
 	"strconv"
 	"strings"
 
+	jsonpatch "github.com/evanphx/json-patch/v5"
```

Comment:
question: I forgot if we cared about camel casing package import alias names. If we do then `jsonPatch`?

Reply:
I don't know either, my IDE did this, and I don't see it as being unreadable